modules/ztp-additional-worker-to-sno-cluster.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-additional-worker-sno"]
+= {sno-caps} cluster expansion with worker nodes
+include::../_attributes/common-attributes.adoc[]
+//:context: additional-worker-sno
+
+You can add one or more worker nodes to an existing SNO cluster to increase CPU resources used by the original SNO control plane node. The addition of worker nodes does not require any downtime for the existing SNO.
+Once you add the worker nodes, the expanded cluster will contain the following nodes:
+
+* The original SNO that serves as the control plane for itself and the additional worker nodes.
+* One or more additional worker nodes that are controlled by the original SNO.
+
+[NOTE]
+====
+Although there is no specified limit on the number of worker nodes that you can add, you must revaluate the reserved CPU allocation on the control plane node for the additional worker nodes.
+====
+
+If workload partitioning is required on the worker node, the policies configuring the worker node must be deployed and remediated before the node installation. This way, the workload partitioning `MachineConfig` objects are rendered and associated with the `worker` `MachineConfig` pool before the `MachineConfig` ignition is downloaded by the installing worker node.
+
+The recommended procedure order is remediating policies, then installing the worker node.
+If workload partitioning manifests are created after the node is already installed, you must manually drain the node and delete all the pods managed by daemonsets. The new pods restarted by their managing daemonsets undergo the workload partitioning process when the pods are created.
```

Comment:
Consider restructuring to avoid passive voice. Often times, when passive voice occurs, the subject of the action is unknown. 

For example: "If x creates workload partitioning manifests after node installation, ..." where x is what is creating the manifests. Similarly, "When the managing daemonsets create the new pods, the new pods undergo the workload partitioning process."